source/2020-05-15-2020-ember-roadmap.md

```diff
@@ -0,0 +1,101 @@
+
+- **Fix router accessibility** so that page navigation is correctly announced by screen readers, without needing a third-party addon.
+- Incorporate **accessibility checks** into the built-in test helpers.
+- **Engage with standards bodies** to help fill the gaps in existing web accessibility APIs.
+
+To contribute to this effort see [RFC Issue 595](https://github.com/emberjs/rfcs/issues/595) which coordinates the Ember A11y Strike Team.
+
+### Sharing Octane outside our community
+
+There are more people building web applications than ever, and Ember must adapt to their changing needs and expectations in order to stay relevant. Octane better aligns Ember's API with what new users expect from a modern framework. We need to take advantage of that change.
+
+This year, we’ll share with the world how Ember Octane is modern, productive, and *fun*. Through blog posts, videos, social media, meetups, and conferences, we will share our knowledge and experiences with the wider JavaScript community and encourage them to give Octane a try.
+
+We will continue to make Octane more attractive to new users with a new documentation approach, more effective website, and with clearer communication about the Glimmer.js project.
+
+---
+
+## Lowering barriers to adoption, and collaboration
+
+Making Ember more attractive to new users doesn't mean compromising on what has made the framework so very successful for existing codebases and teams.
+
+The most basic value of the Ember project is that we solve problems together. While we intend to grow the number of framework users and modernize the framework in many ways, we won't optimize for growth at the expense of our existing community. Instead, we will collaborate on solutions that come with a curated adoption story.
+
+Because we understand and value the power of collaboration, we know we must lower barriers, technical and non-technical, which not only make new users hesitate to adopt Ember but also discourage Ember developers from participating the greater JavaScript community.
+
+A great example of our progress in this is Octane's embrace of native JavaScript classes: A JavaScript developer starting out with Ember today isn't immediately forced to learn a new, Ember-specific class API before they can get to writing code. On the other hand, a developer who starts off with Ember can also contribute to most OSS JavaScript projects without needing to first un-learn the Ember class system.
+
+By sharing common solutions to common problems with other communities we not only make Ember more approachable, we also benefit from the opportunity to exchange more ideas. Everyone wins.
+
+With our efforts to flesh out Octane, improve the build system and align it with the rest of the JavaScript community, raise the baseline support for accessible applications, and to better communicate in 2020 we will lower barriers to adoption of Ember, but also to our own collaboration with the greater JavaScript project.
```

Comment:
@mixonic 
What do you think about closing with a comment for how folks can contact someone if they have comments or questions?